resources/repos.json

```diff
@@ -658,7 +658,7 @@
 		},
 		"options": {
 			"prefix": "DEV",
-			"custom_directory": "J3fftw1/LiteXpansion/master"
+			"custom_directory": "Slimefun-Addon-Community/LiteXpansion/master"
```

Comment:
Now your "custom" directory matches the actual directory, no point in making it "custom" then.
Also, as you change the directory, builds would start at "DEV - 1" again.

Reply:
hmm okay
Then what could causing the litexpansion bug